Content.Server/Cargo/Systems/CargoSystem.Bounty.cs

```diff
@@ -15,6 +16,7 @@
 using Robust.Shared.Containers;
 using Robust.Shared.Random;
 using Robust.Shared.Utility;
+using Serilog;
```

Comment:
This is wrong please remove.

Reply:
Yeah, I was trying to use some logging functions and I think that was accidently added.